worlds/timespinner/__init__.py

```diff
@@ -39,21 +39,19 @@ class TimespinnerWorld(World):
     option_definitions = timespinner_options
     game = "Timespinner"
     topology_present = True
-    data_version = 11
+    data_version = 12
```

Comment:
would probably be better to just remove this?

Reply:
Not sure if we already can remove that?